recipe/meta.yaml

```
@@ -28,17 +28,19 @@ requirements:
   run:
     - anytree >=2.8.0
     - autograd >=1.6.2
-    - casadi >=3.6.3
+    - casadi >=3.6.5
     - imageio >=2.3.0
     - matplotlib-base >=3.6.0
     - numpy >=1.23.5
```

We need an upper pin here as well
```suggestion
    - numpy <=2
```